test/cqlpy/test_materialized_view.py

```diff
@@ -1547,3 +1548,10 @@ def test_view_in_system_tables(cql, test_keyspace):
             assert view in res
             res = [ f'{r.table_name}::{r.index_name}' for r in cql.execute('select * from system."IndexInfo"')]
             assert view.split('.')[1] not in res
+
+# Test view representation in API
+def test_view_in_API(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, "p int PRIMARY KEY, v int") as base:
+        with new_materialized_view(cql, base, '*', 'v,p', 'v is not null and p is not null') as view:
+            res = rest_api.get_request(cql, f"/column_family/built_indexes/{base.replace('.',':')}")
+            assert res not in res
```

Comment:
This looks like a typo, I guess you wanted `view not in res`, not `res not in res`?
Because of this typo this test will always pass (nothing is "in" itself).
I also don't understand the replace() above it.

Reply:
Also... the API call is "built_indexes" - it's probably not the API call you meant to check!